### DISCUSSION TITLE
Guess 🇺🇸 state

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1143] [Galaxy Trivia] [answered]
🏎️🏀🌽
☃️🏒🇸🇪
🦀🐢🥍

**Comment by thuongtruong1009** [comment, marked as answer]
🏎️🏀🌽: Indiana (Known for the Indianapolis 500, basketball, and agriculture)
☃️🏒🇸🇪: Minnesota (Cold winters, hockey, and a large Scandinavian population)
🦀🐢🥍: Maryland (Known for blue crabs, terrapin turtles, and lacrosse)

**Reply by ParanoidUser** [reply]
Bingo! 👏👏
